notifications/util.go

```diff
@@ -27,11 +33,20 @@ func toHeaders(headersArray []string) map[string]string {
 			headersMap[strings.TrimSpace(kv[0])] = ""
 			continue
 		}
-		headersMap[strings.TrimSpace(kv[0])] = strings.TrimSpace(kv[1])
+		headersMap[strings.TrimSpace(kv[0])] = ReplaceEnvVariablePlaceholder(strings.TrimSpace(kv[1]))
 	}
 	return headersMap
 }
 
+func ReplaceEnvVariablePlaceholder(value string) string {
+	envVariableRegex := regexp.MustCompile(EnvVariableRegex)
+	if envVariableRegex.MatchString(value) {
+		envVariableName := value[6 : len(value)-1]
```

Comment:
i know the 6 is the length of the ${env: string but it looks a bit cryptic. maybe you can use a constant with a name that describes this? or even better would be to use groups in regex and capture just the part that we want.

Reply:
I could not find an easy way to capture the matching part in GO, although I find a way to get the prefix. It's more elegant now, let me know if that is ok now or you have something better in mind.
See 1ec34c2